lib/logstash/filters/pilar.rb

```diff
@@ -44,45 +44,56 @@ class Pilar < LogStash::Filters::Base
       # ['(\d+\.){3}\d+'] for IP addresses to be extracted before parsing begins.
       config :regexes, validate: :array, required: false, default: []
 
+      # This determines the maximum size of the single, double, and triple gram dictionaries respectively.
```

Comment:
@aaronabraham311 I've added a new parameter here, how do you want to coordinate updating the ascii docs, should I do it after your first PR is merged in?